### ISSUE TITLE
[SPIKE] Proposal for terms/fields in Service Taxonomy for RO MVP

### ISSUE THREAD

**Issue by davidmpickett** [department-of-veterans-affairs/va.gov-cms#11267]
## Description

**Working document**
- [VBA Services](https://dvagov.sharepoint.com/:x:/s/VBAVA.govmodernization/ET964xEqUFREirhtu34HQVIBbsNOu12QIUc-EVNF_aW98g) (requires VA network)

## Acceptance Criteria
- [x] VBA Services document is reviewed for blockers and decision points
- [x] New tickets created as necessary for remaining work
- [x] A meeting to review is scheduled with Product (UX refinement?)

**Comment by davidmpickett** [comment]
Making this separate from https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11250 just because the review and approval process on the descriptions and whatnot might take longer than that for the terms themselves. So #11250 might be able to move forward a little faster and unblock some things

**Comment by jilladams** [comment]
@davidmpickett is this a dupe of https://github.com/department-of-veterans-affairs/va.gov-team/issues/58511?

**Comment by davidmpickett** [comment]
@jilladams No for a few reasons
1. That ticket is for CAIA to review our work, not for us to do work
2. [The work that we did](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11266) and CAIA reviewed in that ticket was only a portion of the scope. Was limited to only what was absolutely vital for the usability research & prototype, not everything for MVP.
3. CAIA may want to use that same ticket to review any follow up work we do (possibly even this ticket)

**Comment by jilladams** [comment]
@davidmpickett I failed to ask points, but am guessing 5? Update if I'm wrong.

**Comment by davidmpickett** [comment]
We will gain valuable insights into how users understand the taxonomy terms from the usability research. Until that is complete, there's no value in moving forward on this

**Comment by jilladams** [comment]
Unblocking / queuing for S95, since synthesis will wrap in S94.

**Comment by jilladams** [comment]
From planning: timeboxing to 5 for S95, 10/11 will articulate points / tasks for this sprint vs. future.

**Comment by davidmpickett** [comment]
@xiongjaneg @jilladams @mmiddaugh We brought this ticket in pretty light on detail. I just renamed it and added ACs. Take a quick look and lemme know if you have any concerns

**Comment by davidmpickett** [comment]
@xiongjaneg @mmiddaugh 

I just stubbed out a ticket for some new functionality we may want/need in the VA Service taxonomy to support the RO MVP. https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15690

**Comment by jilladams** [comment]
ACs look good to me - pls add estimate when any dust settles on AC feedback from others.

**Comment by davidmpickett** [comment]
Simplified list of VBA Services 
[VBA Services trimmed.xlsx](https://github.com/department-of-veterans-affairs/va.gov-cms/files/12921558/VBA.Services.trimmed.xlsx)

**Comment by davidmpickett** [comment]
## Add to VA Service Taxonomy

These terms reflect the core offerings at regional offices and we have solid understandings of them. These could be a first batch of terms to add into Drupal and discuss with VBA / Taxonomy Council 

- Benefit letters and decision letters
- Burial in a VA cemetery
- Burials and memorials
- Compensation benefits for family members
- Disability compensation
- Education benefits for family members
- Education benefits for Veterans
- Home loans
- Life Insurance
- Military records
- State and local benefits
- Veterans Pension
- Veterans Service Organizations

## Discuss with Facilities team
These terms need some internal conversation/decisions before adding to the taxonomy

- Financial services
- Direct deposit information
- Homeless Veteran care
- Military sexual trauma-related claims
- Veteran Readiness and Employment programs

## Discuss with VBA
These terms may be out of scope for the MVP Pilot phase, or otherwise need some conversations and discussions that will be easier to have once we've gone through an initial batch.

- Careers and employment
- Caregiver support
- Grief and bereavement counseling
- Integrated Disability Evaluation System
- Medical records
- Pre-discharge claims
- Returning service member care
- Transition help
- Veteran profile information
- VetSuccess on Campus

**Comment by mmiddaugh** [comment]
@davidmpickett 
Wondering the distinction between these existing and proposed taxons:

**Existing service: Veteran connections** 
- Vet Center service description: We can help connect you to VA and community resources that will support you in achieving your goals. 

**Proposed service:  State and local benefits** 
- Proposed VBA description: We can help you find and understand the various benefits and services that states and local governments offer Veterans, family members, caregivers, and survivors. And we can help you access those benefits and services.
**Proposed service: Veterans service organizations** 
- Proposed VBA description: We can connect you with a VA-recognized Veterans Service Organization who can file a claim, decision review, or appeal on your behalf and find state and local facilities and programs. 

Vet Center editors enter details about Veteran services organizations and local/state specific resources into their "Veteran connections" local descriptions.

**Comment by davidmpickett** [comment]
> @davidmpickett Wondering the distinction between these existing and proposed taxons:
> 
> **Existing service: Veteran connections**
> 
> * Vet Center service description: We can help connect you to VA and community resources that will support you in achieving your goals.
> 
> **Proposed service: State and local benefits**
> 
> * Proposed VBA description: We can help you find and understand the various benefits and services that states and local governments offer Veterans, family members, caregivers, and survivors. And we can help you access those benefits and services.
> 
> **Proposed service: Veterans service organizations**
> 
> * Proposed VBA description: We can connect you with a VA-recognized Veterans Service Organization who can file a claim, decision review, or appeal on your behalf and find state and local facilities and programs.
> 
> Vet Center editors enter details about Veteran services organizations and local/state specific resources into their "Veteran connections" local descriptions.

Great question. My understanding of "State and Local benefits" vs "Veterans service organizations" is coming from the public contact research where they were described as fundamentally two different types of service.

**State and Local benefits** = Benefits/programs the public contact staff at the regional office _routinely_ help Veterans understand/apply for. Here's a list of benefits in this category that were mentioned in the research.
- annuity
- compensation (state) | chapter 115
- driver's license
- hunting & fishing license
- license plate
- public transit benefits
- reduced taxes (property, home sale)
- toll road pass
- welcome home bonus

**Veterans service organizations** = [accredited representatives](https://www.va.gov/disability/get-help-filing-claim/) who help Veterans and beneficiaries apply for benefits and can advocate for them / give them advice in ways the staff in the Regional Office cannot. They often have offices in the same building as the regional office, but are not VA staff. Having a separate service where ROs could list out the contact information and clarify that Veterans/beneficiaries need to contact those organizations directly/separately. 

**Veteran connections** = Feels blobby. Sounds like it that might have been sufficient for Vet Centers, but I do not think it is a fits what we know about VBA. 

All that being said, seems like there may be more conversation to have about these two and they drop out of the first bucket into one of the others.

**Comment by mmiddaugh** [comment]
The intended meaning/purpose for **Veteran connections** has felt nebulous to editors - at least in name. 

This might be the opportunity to either evolve that particular service or to allow non-VBA editors to leverage these new services.

**Comment by davidmpickett** [comment]
Just stubbed a ticket for [change management around Veteran connections](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15700) for our future selves. This is part of a [new epic](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15698) I made to group all the tickets related to term changes

**Comment by davidmpickett** [comment]
Started a branch / PR to create some runbooks for taxonomy terms: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/15727

Here's a sneak peek of the Add Term runbook - https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/va-service-taxonomy/add-term-runbook.md

**Comment by davidmpickett** [comment]
New issue template for adding a term to taxonomy! https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/.github/ISSUE_TEMPLATE/taxonomy-add-term.yml

**Comment by davidmpickett** [comment]
@jilladams @xiongjaneg @mmiddaugh
I think I am at the finish line for this one. There are lots of follow-up tickets I created to try and chart the path forward for VBA RO MVP and for the Service Taxonomy as a product. 
 
## EPIC: Terms to review [#15698](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15698) 
- Possible Sprint 96 candidate? - #15748
- #15751
- #15700
- I haven't covered all the terms for VBA with these tickets, but I think it's a good start
## EPIC: Service Taxonomy enhancements [#13140](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13140)
- Important for VBA - #15690
- Important for Facility Locator - #15685
- Medium importance #15695
- Low importance #13139 
- Very low importance #15684
